flathub.json

```
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["aarch64"]
```

this is disappointing. why not building from sources?

---

There are a lot things f-e-d-c doesn't do well anyway.

As you just said this would make aarch64 packages and that's the idea.

---

```suggestion
  "only-arches": ["x86_64"]
```

This is a better way to write it. Still disappointed that you don't want to do it.

---

I do want to do it, just not right at this moment since it would require more work and I'd like to get any flatpak at all released first. At least 88 people have voted for the [upstream issue](https://github.com/johannesjo/super-productivity/issues/25) requesting this package, but as far as I know there haven't been any requests for aarch64 native binary packages for Linux.